src/screens/Authentication/Authentication.stories.tsx

```diff
@@ -52,6 +53,15 @@ export const Welcome = {
   ),
 };
 
+export const PreSetup = {
+  parameters: withFigmaDesign(
+    "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=304-317931&t=3EAIRe8423CpOQWY-4"
+  ),
+  args: {
+    isSetup: true,
```

Comment:
Shouldn't this be `false` for a `PreSetup` story? Or rename the story to `PostSetup` or something.

Reply:
I guess maybe I'm confused as to the meaning of `isSetup`. Does it mean "the project has been setup" or "this is the setup screen" (i.e. the project _hasn't_ been setup)? It seems to be the former. Maybe change the prop name to avoid that ambiguity.

Reply:
"PreSetup" is short for "Previously Setup", I'll change it.
"isSetup" means there is a a projectId present, meaning someone has done that setup work already previously.

I'll change both to "hasProjectId", do you feel that avoids ambiguity?